opm/output/eclipse/WriteInit.cpp

```diff
@@ -673,24 +908,89 @@ void Opm::InitIO::write(const ::Opm::EclipseState&              es,
         if (es.globalFieldProps().has_double("MULTPV")) {
             multipliers.insert("MULTPV", UnitSystem::measure::identity,
                         es.globalFieldProps().get_double("MULTPV"),
-                        data::TargetType::INIT);
+                        data::TargetType::INIT); 
         }
 
         writeSimulatorProperties(grid, multipliers, initFile);
     }
 
-    writeTableData(es, units, initFile);
+    // ITEM 3 - LOOP FOR LOCAL PROPERTIES LGR
+    if (grid.is_lgr()) 
+    {
```

Comment:
We generally prefer to place the opening brace on the same line as the condition.